openqa-qam-sle

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env bash
+
+# Search openQA jobs related to a given update inside job groups:
+# - Maintenance: Single Incidents / Core Incidents
+# - Maintenance: Aggregated updates / Core Maintenance Updates
+# Default search within the last 5 days for "Aggregated updates" builds
+# Usage: eg
+#   openqa-qam-sle -a SUSE:Maintenance:34576:337335
+#   openqa-qam-sle -a SUSE:Maintenance:34577:337331
+#   openqa-qam-sle -a SUSE:Maintenance:34564:337279
+#   openqa-qam-sle -a SUSE:Maintenance:34581:337626
```

Comment:
Maybe the usage function should print that. So it would make sense to turn this comment into a multi-line string and print it in the usage function. You can move the usage function to the very top so this can still be easily found when looking at the script code itself.

Reply:
Done